block-node/verification/src/main/java/org/hiero/block/node/verification/session/HapiVersionSessionFactory.java

```diff
@@ -28,20 +27,19 @@ private BlockVerificationSessionFactory() {}
      * @throws NullPointerException if blockSource or hapiVersion is null
      * @throws IllegalArgumentException if blockNumber less than 0 or version unsupported
      */
-    public static BlockVerificationSession createSession(
+    public static VerificationSession createSession(
             final long blockNumber, final BlockSource blockSource, final SemanticVersion hapiVersion) {
-
         Objects.requireNonNull(blockSource, "blockSource cannot be null");
         Objects.requireNonNull(hapiVersion, "hapiVersion cannot be null");
         Preconditions.requireWhole(blockNumber, "blockNumber must be >= 0");
 
         if (isGreaterThanOrEqual(hapiVersion, V_0_68_0)) {
-            return new ExtendedMerkleTreeVerificationSessionV0680(blockNumber, blockSource, "extraBytesPlaceholder");
+            return new ExtendedMerkleTreeSession(blockNumber, blockSource, "extraBytesPlaceholder");
         } else if (isGreaterThanOrEqual(hapiVersion, V_0_64_0)) {
-            return new PreviewSimpleVerificationSessionV0640(blockNumber, blockSource);
+            return new PreviewSimpleHashSession(blockNumber, blockSource);
         } else {
-            throw new IllegalArgumentException("Unsupported HAPI version: %s (supported from 0.64.0 and up)"
-                    .formatted(toShortString(hapiVersion)));
+            final String unsupportedMessage = "Unsupported HAPI version: %s (supported from 0.64.0 and up)";
+            throw new IllegalArgumentException(unsupportedMessage.formatted(toShortString(hapiVersion)));
```

Comment:
Per a previous note should we be using string replacement here instead of the more costly formatted?